util/otbn_build.py

```diff
@@ -200,9 +200,12 @@ def main() -> int:
         sym_pfx = '_otbn_app_{}_'.format(app_name)
         out_embedded_obj = out_dir / (app_name + '.rv32embed.o')
         args = (['-O', 'elf32-littleriscv',
-                 '--prefix-sections=.rodata.otbn',
                  '--set-section-flags=*=alloc,readonly',
                  '--set-section-flags=.data=load',
+                 '--rename-section=.text=.rodata.otbn.text',
+                 '--rename-section=.start=.rodata.otbn.start',
+                 '--rename-section=.data=.rodata.otbn.data',
+                 '--rename-section=.bss=.bss.otbn.bss',
```

Comment:
Hmm, why is this needed?

Reply:
Ah, I see. Presumably the problem is that if the OTBN BSS ends up mixed in with all the other rodata then it takes up address space (even though it's NOLOAD). Probably the resulting ELF has a gap between the segments, but that doesn't help because it has to be zero-padded when generating the Ibex bin file. (Is that right?)

Have you tried making the .bss section not allocatable? I think this might be an easier approach, that also won't require Ibex to zero out a big wodge of its own memory at runtime.

Reply:
> Ah, I see. Presumably the problem is that if the OTBN BSS ends up mixed in with all the other rodata then it takes up address space (even though it's NOLOAD). Probably the resulting ELF has a gap between the segments, but that doesn't help because it has to be zero-padded when generating the Ibex bin file. (Is that right?)

Yes, that's exactly right. Everything that ends up in the rodata section ends up getting loaded, including the OTBN BSS if it's there, and the binary size doesn't decrease.

> Have you tried making the .bss section not allocatable? I think this might be an easier approach, that also won't require Ibex to zero out a big wodge of its own memory at runtime.

I tried setting the flags so `.bss` is not allocatable, but it still seems to be allocated in the resulting binary based on the size. For clarity, I changed the arguments here to:
```
                 '--set-section-flags=*=alloc,load,readonly',
                 '--set-section-flags=.bss=noload,readonly',
                 '--rename-section=.text=.rodata.otbn.text',
                 '--rename-section=.start=.rodata.otbn.start',
                 '--rename-section=.data=.rodata.otbn.data',
                 '--rename-section=.bss=.rodata.otbn.bss',
                 '--remove-section=.scratchpad',
```
Is that what you meant?

Reply:
I think that `--prefix-sections=.rodata.otbn` might be easier, but let's worry about that in a follow-up PR :-)